WPMedia/fusion-news-theme-blocks
# TMEDIA-732: Byline

## Description

Add Byline block for news media

## Jira Ticket

- [TMEDIA-732](https://arcpublishing.atlassian.net/browse/TMEDIA-732)

## Acceptance Criteria

Convert Byline Block to 2.0

## Test Steps

1. Checkout this branch `git checkout tmedia-732-byline`
2. Run `npm i` in `arc-themes-feature-pack`
3. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/byline-block`
4. Add Byline Block to a page
5. Use content-api id: `CYZDXGIOZ5BLXL6TOTPO5VQHJQ`
6. Run storybook and check documentation: `npm run storybook`

## Dependencies or Side Effects

https://github.com/WPMedia/arc-themes-feature-pack/pull/259
https://github.com/WPMedia/arc-themes-manifests/pull/59

## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
